db/python/layers/seqr.py

```diff
@@ -114,6 +130,7 @@ async def sync_dataset(
         sync_individual_metadata: bool = True,
         sync_individuals: bool = True,
         sync_es_index: bool = True,
+        es_index_types: list[str] = None,
```

Comment:
The values must be known to avoid key errors, so using an enum here is best IMO. Then defining it once, and putting it on the API means you can reference it from the UI (and one change here automatically populates the select grid).